docs/pages/eas/workflows/examples/deploy-to-production.mdx

```diff
@@ -83,6 +83,9 @@ jobs:
   fingerprint:
     name: Fingerprint
     type: fingerprint
+    # @info Use the same environment as your build profile #
+    environment: production
+    # @end #
```

Comment:
```suggestion
    environment: production
```

Nothing else is highlighted in this example, and emphasizing only `environment` doesn't make sense in this context.